pages/api/content/[type].js

```diff
@@ -107,17 +109,10 @@ export default async (req, res) => {
 
   // Group filtering if not set marks it to null
   const filterParams = {
-    groupId: groupId ? groupId : null,
-    groupType: groupType ? groupType : null,
-  }
-
-  if (author) {
-    filterParams.author = author
-  }
-
-  if (tags) {
-    // TODO: Make tags filter generic
-    filterParams['tags.slug'] = tags
+    ...(groupId ? { groupId } : {}),
```

Comment:
it is a must that groupId is set to null, when we are filtering content for the general site we want content that has no group, ergo groupId must be a `null`, same with groupType. 
the rest is fine

Reply:
then I think I have to change condition to `groupId !== undefined`